.github/workflows/ci.yaml

```diff
@@ -37,6 +37,10 @@ jobs:
           yarn install
           yarn global add react-scripts
 
+      - name: Run linter
```

Comment:
One option would be to use https://github.com/marketplace/actions/lint-action this will leave comments on the PR.